test/jest/get-lockfile-version.test.ts

```diff
@@ -0,0 +1,10 @@
+import { getLockfileVersionFromFile } from '../../lib/utils';
+
+describe('getLockfileVersionFromFile', () => {
+  it('npm', () => {
+    const result = getLockfileVersionFromFile(
+      `${__dirname}/../fixtures/bare-npm/package-lock.json`,
+    );
+    expect(result).toMatchSnapshot();
```

Comment:
This Snapshot should be failing, the lockfile version is 2 but the snapshot says NPM_LOCK_V1

Seems the function is behaving not as expected